### PR TITLE
executor: add benchmark for sortExec

### PR DESCRIPTION
**Pull request by SunRunAway**
### What problem does this PR solve? 

add benchmark for `sortExec`

### Check List 

Tests 

 - None

Code changes

 - None

Side effects

 - None

Related changes

 - None

Release note

 - None

### CODE REVIEW

File: executor/benchmark_test.go [hunk 1]
```diff
@@ -691,3 +691,99 @@ func BenchmarkBuildHashTableForList(b *testing.B) {
...
+
+func BenchmarkSortExec(b *testing.B) {
+	b.ReportAllocs()
+	cas := defaultSortTestCase()
+	cas.ndvs = []int{0, 0} // all random data
+	b.Run(fmt.Sprintf("%v", cas), func(b *testing.B) {
+		benchmarkSortExec(b, cas)
+	})
+
+	cas.ndvs = []int{4450, 0} // values of the first column are mostly equal
```
Comment (zz-jason) [inline executor/benchmark_test.go]: seems this benchmark is the same as the first one? they both have no order by columns.

Comment (qw4990) [inline executor/benchmark_test.go]: Why a fixed NDV 4450?
We can use a loop like `for ndv := 1; ndv < 10000000; ndv *= 2` to see it's performance on different NDVs.

**Comment by SunRunAway** [comment]
Abandon, will file another PR.

**Pull request closed without merging**